src/index.js

```diff
@@ -9,6 +9,9 @@ const crypto = require('libp2p-crypto')
 const assert = require('assert')
 const waterfall = require('async/waterfall')
 
+const protons = require('protons')
+const {PeerIdProto} = protons('message PeerIdProto { required bytes id = 1; bytes pubKey = 2; bytes privKey = 3; }')
```

Comment:
Perhaps add this to a separate file? Looks weird here.

Reply:
I think I should, was just lazy because it's so small ;)